### DISCUSSION TITLE
Example.cs does not work

### DISCUSSION THREAD

**Discussion by hsnprojects** [tui-cs/Terminal.Gui#3578] [Q&A] [answered]
Example.cs app runs, but after logging in it cannot retrieve usernameText field because Application.Top is null.

**Comment by tig** [comment]
I can't repro this on v2_develop. 

Is this v1?

**Comment by tig** [comment, marked as answer]
Fixed in #3580 

Thanks!

**Comment by hsnprojects** [comment]
I did not realize that v2_develop is the current version. I will check out that branch!

**Reply by dodexahedron** [reply]
Well, it's "current" in that it's the newest and what is being actively developed. It has not yet reached stable release though. Just a heads up.

But we're closing in on it!
